client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/steps/step-add-migration-key-fallback.tsx

```diff
@@ -9,11 +10,19 @@ export const StepAddMigrationKeyFallback: FC = () => {
 	const translate = useTranslate();
 	const site = useSite();
 	const siteUrl = site?.URL ?? '';
+	const isWhiteLabeledPluginEnabled = config.isEnabled(
+		'migration-flow/enable-white-labeled-plugin'
+	);
+	const migrationKeyLabel = isWhiteLabeledPluginEnabled
+		? 'Migration Key'
+		: 'Migrate Guru Migration Key';
+	const migrateLabel = isWhiteLabeledPluginEnabled ? 'Start migration' : 'Migrate';
```

Comment:
Are you planning to add the String Freeze label and waiting for translations? Because otherwise we need to check that we have translations first, right?

Reply:
Right, thanks for the reminder. I'll add the checks so we can ship it faster. I don't think shipping with the old strings will be a big deal.

Reply:
I've added the translation checks where it makes sense. Some strings like `Start migration` are not translated on the BlogVault screen so I'm skipping them.

Reply:
Makes sense. When I switched the language though, I see we are still mentioning Migrate Guru. We're going to need to remove all mentions of it since we're installing the other plugin now:

![Screenshot 2024-10-03 at 12 27 43 PM](https://github.com/user-attachments/assets/d56cf673-65ad-468b-a583-5cd9be3e592c)

Reply:
Sorry to pile on to what Dale mentioned below, but there weren't any testing instructions for testing this fallback text. I manually edited some code to enable this state.

Reply:
> there weren't any testing instructions for testing this fallback text

Oh, forgot to mention that one, sorry.

I'm not sure if the fallback is still a thing. While testing, I wasn't able to get to that state. For now, I decided to keep it and ask Mark when he's back if it's needed. "Don't take a fence down until you know the reason why it was put up" (or something like that 😅).

> When I switched the language though, I see we are still mentioning Migrate Guru

Interesting. I'll investigate...

Reply:
> When I switched the language though, I see we are still mentioning Migrate Guru

This is an expected behavior because the [translation](https://github.com/Automattic/wp-calypso/pull/95101/files#diff-f9552f1cb17d596dac445265d8e14e44b808c175b287fea6eac1ee5b5161d087R38-R39) is still missing and the [fallback translation](https://github.com/Automattic/wp-calypso/pull/95101/files#diff-f9552f1cb17d596dac445265d8e14e44b808c175b287fea6eac1ee5b5161d087R49) is displayed.

Reply:
Thanks for the review, Donna! I've added a card for the cleanup. It's on the fallback component so not many people will see it.